src/tokens/TabWallet.tsx

```diff
@@ -1,231 +1,83 @@
-      transform: [
-        {
-          translateY:
-            scrollPosition.value > nonStickyHeaderHeight
-              ? -nonStickyHeaderHeight
-              : -scrollPosition.value,
-        },
-      ],
-      shadowColor: interpolateColor(
-        scrollPosition.value,
-        [nonStickyHeaderHeight - 10, nonStickyHeaderHeight + 10],
-        ['transparent', Colors.gray1]
-      ),
-    }
-  }, [scrollPosition.value, nonStickyHeaderHeight])
-
-  const animatedFooterStyles = useAnimatedStyle(() => {
-    return {
-      transform: [
-        {
-          translateY: footerPosition.value,
-        },
-      ],
-    }
-  }, [footerPosition.value])
-
-  useScrollAwareHeader({
-    navigation,
-    title: t('bottomTabsNavigator.wallet.title'),
-    scrollPosition,
-    startFadeInPosition: nonStickyHeaderHeight - HEADER_OPACITY_ANIMATION_START_OFFSET,
-    animationDistance: HEADER_OPACITY_ANIMATION_DISTANCE,
-  })
-
-  const handleMeasureNonStickyHeaderHeight = (event: LayoutChangeEvent) => {
-    setNonStickyHeaderHeight(event.nativeEvent.layout.height)
-  }
-
-  const handleMeasureListHeaderHeight = (event: LayoutChangeEvent) => {
-    setListHeaderHeight(event.nativeEvent.layout.height)
-  }
-
-  const handleMeasureListFooterHeight = (event: LayoutChangeEvent) => {
-    setListFooterHeight(event.nativeEvent.layout.height)
-  }
-
-  const handleChangeActiveView = (selectedTab: AssetTabType) => {
-    navigation.setParams({ activeAssetTab: selectedTab })
-  }
-
+  const totalTokenBalanceLocal = useTotalTokenBalance()
+  const balanceDisplay = hideWalletBalances
+    ? `XX${decimalSeparator}XX`
+    : totalTokenBalanceLocal?.toFormat(2)
   return (
-    // Transparency issue on Android present when a fragment is used - Nested Animated.View prevents it
-    <Animated.View>
-      <Animated.View
-        style={[styles.listHeaderContainer, animatedListHeaderStyles]}
-        onLayout={handleMeasureListHeaderHeight}
-      >
-        <View
-          style={[styles.nonStickyHeaderContainer]}
-          onLayout={handleMeasureNonStickyHeaderHeight}
-        >
-          <View style={styles.balanceSection}>
-            <AssetsTokenBalance showInfo={false} />
-          </View>
-          <ActionsCarousel key={'ActionsCarousel'} />
-        </View>
-        <AssetTabBar
-          activeTab={activeTab}
-          onChange={handleChangeActiveView}
-          displayPositions={displayPositions}
-        />
-      </Animated.View>
-
-      <AssetList
-        activeTab={activeTab}
-        listHeaderHeight={listHeaderHeight}
-        handleScroll={handleScroll}
-      />
-      {showClaimRewards && (
-        <Animated.View
-          style={[styles.footerContainer, animatedFooterStyles]}
-          onLayout={handleMeasureListFooterHeight}
-        >
-          <Button
-            type={BtnTypes.SECONDARY}
-            size={BtnSizes.FULL}
-            text={t('assets.claimRewards')}
+    <SafeAreaView style={styles.container} edges={['top']}>
```

Comment:
Used `SafeAreaView` but looks like it is adding too much space at the top? Wasn't sure if that was an issue or just what it should look like on this size iPhone (as opposed to the smaller screen in designs).

Reply:
Try edges={[]}

You'll need to add a bit of padding, but that should remove that empty space at the top

Reply:
Do you need a SafeAreaView with only top edge (or no edge)? This is not touching the top edge of the screen right?

Reply:
![Simulator Screenshot - iPhone 15 Pro - 2024-11-05 at 13 52 47](https://github.com/user-attachments/assets/331ee208-67b2-4d2b-925b-94fd096bdd92)
just using a View seems to be okay